src/pipecat/services/simli/video.py

```diff
@@ -137,19 +152,47 @@ async def process_frame(self, frame: Frame, direction: FrameDirection):
 
                 resampled_frames = self._simli_resampler.resample(old_frame)
                 for resampled_frame in resampled_frames:
-                    await self._simli_client.send(
-                        resampled_frame.to_ndarray().astype(np.int16).tobytes()
-                    )
+                    audioBytes = resampled_frame.to_ndarray().astype(np.int16).tobytes()
+                    if self._previously_interrupted and self._is_gs_avatar:
```

Comment:
I believe you don't need to check `self._is_gs_avatar` here, since `self._previously_interrupted` can only be `True` if `self._is_gs_avatar` is, too...right?

Reply:
I believe this hasn't been resolved yet